src/SwiftUI/SwiftUI/Interop/CustomViewType.cs

```diff
@@ -60,7 +60,8 @@ internal CustomViewType (Type customViewType): base (customViewType, MetadataKin
 				// Currently unused, so don't force allocation if it's a custom view
 				//thunkMetadata->ThunkViewTViewConformance = swiftBodyType.ViewConformance;
 				thunkMetadata->ThunkViewTViewConformance = null;
-			} catch {
+			}
+			catch {
```

Comment:
Let's avoid adding this extra whitespace

Reply:
Done!